brave/brave-core
# (Android) Redesign private tab UI

## Resolves https://github.com/brave/brave-browser/issues/37596

Redesign private tab UI following [Figma design](https://www.figma.com/design/jt7tV0UsU7XhJXR2B8807E/Private-and-Tor-Windows---Nala?node-id=37%3A7636&t=wKzZwmhqCGW5ylK4-1).

Private tab redesign includes three new sections "Leave no trace", "Browsing visibility" and "Brave VPN".
Brave VPN section will show a call to action text view that will redirect to the subscription page. If the user is already a paying customer, or the VPN is not supported the CTA view will be hidden.

The design is responsive and adjust to large device screens and tablets, following the logic provided by Figma design.
Under normal conditions the entire layout is centered in the visible part of the page.

### Smartphone

| Layout Bounds OFF | Layout Bounds ON |
| ------------- |- |
| ![Screenshot 2024-10-15 at 4 18 21 PM](https://github.com/user-attachments/assets/9a8b4db6-2f70-4e01-8b02-ad9e302e803a)  | ![Screenshot 2024-10-15 at 4 57 14 PM](https://github.com/user-attachments/assets/7e40f274-53a7-4160-8720-4a7562ab47b1) |


### Tablet

| Layout Bounds OFF | Layout Bounds ON |
| ------------- |- |
| ![Screenshot 2024-10-15 at 5 02 30 PM](https://github.com/user-attachments/assets/f576eb7a-fc83-4357-a63e-562f22b8643e)  | ![Screenshot 2024-10-15 at 5 02 39 PM](https://github.com/user-attachments/assets/71fddc95-8ae8-40c0-b51b-fa4484e4b8c9) |



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Tap on the overflow menu icon
- Tap on New private tab
- Observe the private UI loads the new layout containing three blocks
- Observe the new layout is loaded centered in the visible part of the page
- Tap on "Try free for 7 days"
- Observe the VPN subscription page loads correctly
